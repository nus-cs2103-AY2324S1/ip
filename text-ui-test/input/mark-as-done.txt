todo do task 1
todo do task 2
list
todo task3
mark 2
list
mark 3
list
unmark 2
list
mark 4
list
unmark 6
list
bye