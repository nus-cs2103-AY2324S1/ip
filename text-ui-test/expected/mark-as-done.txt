____________________________________________________________
Loading data from hard disk ...
Done loading.
____________________________________________________________
Hello from Quack-NKN
What can I do for you?
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] do task 1
Now you have 1 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] do task 2
Now you have 2 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][ ] do task 2
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] task3
Now you have 3 in the list.
____________________________________________________________
In: ____________________________________________________________
Nice! I've marked this task as done:
[T][X] do task 2
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][X] do task 2
3. [T][ ] task3
____________________________________________________________
In: ____________________________________________________________
Nice! I've marked this task as done:
[T][X] task3
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][X] do task 2
3. [T][X] task3
____________________________________________________________
In: ____________________________________________________________
OK, I've marked this task as not done yet:
[T][ ] do task 2
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][ ] do task 2
3. [T][X] task3
____________________________________________________________
In: ____________________________________________________________
Quack, the task number you provide does not exist!
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][ ] do task 2
3. [T][X] task3
____________________________________________________________
In: ____________________________________________________________
Quack, the task number you provide does not exist!
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
2. [T][ ] do task 2
3. [T][X] task3
____________________________________________________________
In: ____________________________________________________________
Bye. Hope to see you again soon!
____________________________________________________________